patches/tModLoader/Terraria.ModLoader/MonoModHooks.cs

```diff
@@ -15,6 +15,24 @@ namespace Terraria.ModLoader
 {
 	public static class MonoModHooks
 	{
+		private static Dictionary<Type, string> defaultAliases = new Dictionary<Type, string> {
```

Comment:
Huh, interesting how this is much simpler than what I can find on Google to get names of primitives.. Nice job